jupyter_book/page.py

```diff
@@ -110,6 +118,10 @@ def build_page(path_ntbk, path_html_output, path_media_output=None, execute=Fals
     exp = HTMLExporter(template_file=path_template, config=c)
     html, resources = exp.from_notebook_node(ntbk, resources=output_resources)
 
+    # Wrap HTML in a {% raw %} tag to prevent errors when a notebook contains
+    # characters that have special meaning for Jekyll (like `{{`).
+    html = HTML_WRAPPER.format(html)
```

Comment:
For example, here we could do a quick find/replace with `{{` and `}}` to make it `{% raw %}{{{% endraw %}`. Would that work?